wolk/model/unit.py

```diff
@@ -0,0 +1,227 @@
+"""Enumeration of available units on the Platform."""
```

Comment:
Can we this file generate from platform and implement import mehanism for connectors? Beacuse it will chnage during time and instance.